src/mon/PGMap.cc

```diff
@@ -3348,9 +3348,13 @@ void PGMap::get_health_checks(
       // application metadata is not encoded until luminous is minimum
       // required release
       if (pool.application_metadata.empty() && !pool.is_tier()) {
-        stringstream ss;
-        ss << "application not enabled on pool '" << pool_name << "'";
-        detail.push_back(ss.str());
+        const utime_t now = ceph_clock_now();
```

Comment:
Might want to make the use of `CLOCK_REALTIME` explicit with `ceph::real_clock::now()`.